packages/ui/src/components/templates/Dashboard/Chart/PieChart.tsx

```diff
@@ -109,38 +136,46 @@ const PieChart = ({
   );
 };
 
-const styles = StyleSheet.create({
-  container: {
-    flexDirection: 'column',
-    justifyContent: 'center',
-    alignItems: 'center',
-    alignContent: 'center',
-    minWidth:
-      Dimensions.get('window').width > 500
-        ? Dimensions.get('window').width / 4 - MARGIN * 2
-        : Dimensions.get('window').width / 2 - MARGIN * 2,
-    margin: MARGIN,
-  },
-  title: {
-    alignSelf: 'center',
-    textAlign: 'center',
-  },
-  legendContainer: {
-    flexDirection: 'column',
-  },
-  itemLegendContainer: {
-    flexDirection: 'row',
-    marginVertical: 5,
-  },
-  legend: {
-    marginHorizontal: 5,
-  },
-  legendColor: {
-    borderWidth: 5,
-    marginVertical: 2,
-    borderTopRightRadius: 7,
-    borderBottomRightRadius: 7,
-  },
-});
+const getStyles = (innerRadius: number) =>
+  StyleSheet.create({
+    centeredView: {
+      justifyContent: 'center',
+      alignItems: 'center',
+      maxWidth: innerRadius * 2,
+      maxHeight: innerRadius * 2,
+      padding: 10,
+    },
+    container: {
+      flexDirection: 'column',
+      justifyContent: 'center',
+      alignItems: 'center',
+      alignContent: 'center',
+      minWidth:
+        Dimensions.get('window').width > 500
+          ? Dimensions.get('window').width / 4 - MARGIN * 2
+          : Dimensions.get('window').width / 2 - MARGIN * 2,
+      margin: MARGIN,
+    },
+    title: {
+      alignSelf: 'center',
+      textAlign: 'center',
+    },
+    legendContainer: {
+      flexDirection: 'column',
+    },
+    itemLegendContainer: {
+      flexDirection: 'row',
+      marginVertical: 5,
+    },
+    legend: {
+      marginHorizontal: 5,
+    },
+    legendColor: {
+      borderWidth: 5,
+      marginVertical: 2,
+      borderTopRightRadius: 7,
+      borderBottomRightRadius: 7,
```

Comment:
It could be nice to split this styles 
Maybe by splitting file in different components (for the legend for example or the label)